src/constants/lights.ts

```diff
@@ -1,7 +1,7 @@
 import { MathUtils } from "three";
 
 const spotlightSettings = Object.freeze({
-  angle: 6 * MathUtils.DEG2RAD,
```

Comment:
This fixes an unrelated visual artifact @toloudis and I found while looking at a bunch of isosurfaces:
![image](https://github.com/user-attachments/assets/72cdab28-0a80-4066-8c0d-58095a663ed0)